### PR TITLE
updated the \README.md file to describe the \ repo.

### PR DESCRIPTION
**Pull request by Raulp8**
final step of github setup

### CODE REVIEW

File: walkthroughs/week-1-setup/intro-walkthrough.md [hunk 1]
```diff
@@ -125,7 +125,6 @@ get more specific instructions each week.
 You finished this introduction!
 
 Next, you'll write HTML and JavaScript to create a portfolio page, and you'll
-deploy it to App Engine so anyone can visit it.
```
Comment (aditij113) [inline walkthroughs/week-1-setup/intro-walkthrough.md:128]: is deleting this line an accident? :)

**Review by aditij113 — CHANGES_REQUESTED** [review]
Nice! Can you attach screenshots of what the page looks like?

File: walkthroughs/week-1-setup/intro-walkthrough.md (hunk 1 above)
Comment (Raulp8) [reply walkthroughs/week-1-setup/intro-walkthrough.md:128]: Screenshot 2020-05-29 at 5 30 50 PM - Display 1
Screenshot 2020-05-29 at 5 31 14 PM - Display 1
yeah, sorry is there an easy way to undo it?

Comment (aditij113) [reply walkthroughs/week-1-setup/intro-walkthrough.md:128]: I think you'll just have to type out the line again :)

**Comment by aditij113** [comment]
Screenshots look good!

**Review by aditij113 — APPROVED** [review]

**Pull request merged**